std/assembly/array.ts

```diff
@@ -111,7 +111,7 @@ export class Array<T> {
     return value;
   }
 
-  @unsafe @operator("{}") private __unchecked_get(index: i32): T {
+  @unsafe @operator("{}") private __uget(index: i32): T {
```

Comment:
that's why latest ts-node turn off cache completely

Reply:
I've encountered the cache problem earlier with `__unchecked_get`, and just then decided to unify the name here. From what I can tell, the ts-node caching stuff typically goes haywire as soon as `CommonFlags`, `FlowFlags` or `LocalFlags` are changed with new values.

Reply:
In this particular case, the problem seems to occur when the value of `FlowFlags.UNCHECKED_CONTEXT` changes. This one just so happens to be the last flag in that enum, so is pretty much always affected. Good indicator I guess :)